source/common/http/http1/codec_impl.cc

```diff
@@ -1273,7 +1276,14 @@ ClientConnectionImpl::ClientConnectionImpl(Network::Connection& connection, Code
                                            ConnectionCallbacks&, const Http1Settings& settings,
                                            const uint32_t max_response_headers_count)
     : ConnectionImpl(connection, stats, settings, MessageType::Response, MAX_RESPONSE_HEADERS_KB,
-                     max_response_headers_count) {}
+                     max_response_headers_count),
+      owned_output_buffer_(connection.dispatcher().getWatermarkFactory().createBuffer(
+          [&]() -> void { this->onBelowLowWatermark(); },
+          [&]() -> void { this->onAboveHighWatermark(); },
+          []() -> void { /* TODO(adisuissa): handle overflow watermark */ })) {
+  // Inform parent
```

Comment:
Set the watermark as done in https://github.com/envoyproxy/envoy/pull/20629/files#diff-8d3b9d0a839b05612561930ee8c58bf1b0ff0adc55cf27d79a7e3bbfbd81965eR969